mkosi/__init__.py

```diff
@@ -7546,9 +7546,21 @@ def find_ovmf_vars() -> Path:
     die("Couldn't find OVMF UEFI variables file.")
 
 
+def qemu_check_kvm_support() -> bool:
+    kvm = Path("/dev/kvm")
+    if not kvm.is_char_device():
+        return False
+    # some CI runners may present a non-working KVM device
+    try:
+        with kvm.open("r+b"):
```

Comment:
The `b` flag actually doesn't have an effect on Linux, so we can omit it. (see https://unix.stackexchange.com/questions/429969/why-opening-file-for-binary-access-is-meaningless-on-unix-nor-the-for-bot)

Reply:
It does make a difference in Python, though.
```
In [1]: from pathlib import Path

In [2]: bar = Path("bar")

In [3]: with bar.open("r+") as f:
   ...:     foo = f.read()

In [4]: foo
Out[4]: 'foo\n'

In [5]: with bar.open("r+b") as f:
   ...:     foo = f.read()

In [6]: foo
Out[6]: b'foo\n'
```
When opened with `b` Python will read bytes instead of strings, which won't attempt any decoding.

We don't really want to read anything here, but I'd gather that in the end this is more defensive.

Reply:
Ahh, in that case we can keep the flat yeah since we don't care about python's text processing